addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -1138,6 +1138,54 @@ QUnit.test('delete all attachments of a message with tracking fields should stil
     );
 });
 
+QUnit.test('add a partner mention after an email + break line', async function (assert) {
+    assert.expect(1);
```

Comment:
Excellent unit test with single assertion!! In such cases, the name of test should almost match the single assertion comment.